src/main/java/com/conveyal/gtfs/api/graphql/StopTimeFetcher.java

```diff
@@ -16,13 +28,99 @@
  * Created by matthewc on 3/9/16.
  */
 public class StopTimeFetcher {
+    public static List<WrappedGTFSEntity<StopTime>> apex(DataFetchingEnvironment env) {
```

Comment:
I still feel kinda weird about having stop times at the top level of the API. Remind me the use case for this?